README.md

```diff
@@ -44,6 +44,17 @@ unless real values don't help users know what to change.
 -->
 
 ```hcl
+data "ibm_iam_auth_token" "auth_token" {}
+
+provider "restapi" {
+  uri = "https://resource-controller.cloud.ibm.com"  # https://private.resource-controller.cloud.ibm.com for private
+  headers = {
+    Authorization  = data.ibm_iam_auth_token.auth_token.iam_access_token
+    "Content-Type" = "application/json"
+  }
+  write_returns_object = true
+}
+
```

Comment:
can you update the usage below to include `app_config_crn` since `cspm_enabled` isa true by default (meaning its required by default)